File: trunk/server/net/Protocols/CardGroupInfo.cs
using ProtoBuf;
using System.Collections.Generic;
using System;

namespace protocol
{
	[ProtoContract]
	public class CardGroupInfo
	{
        /// <summary>
        ///  id
        /// </summary>
		[ProtoMember(1, IsRequired = false)]
		public string id;
        /// <summary>
        ///  属于哪个玩家
        /// </summary>
		[ProtoMember(2, IsRequired = false)]
		public string pid;
        /// <summary>
        ///  卡组名称
        /// </summary>
		[ProtoMember(3, IsRequired = false)]
		public string name;
        /// <summary>
        ///  基地
        /// </summary>
		[ProtoMember(4, IsRequired = false)]
		public string baseInitId;
        /// <summary>
        ///  卡牌组
        /// </summary>
		[ProtoMember(5, IsRequired = false)]
		public List<string> cardIds;

	}
}

File: trunk/server/src/main/java/org/ngame/protocol/domain/CardGroupInfo.java
/**
 * 卡牌
 */
package org.ngame.protocol.domain;



import java.util.List;

import com.baidu.bjf.remoting.protobuf.FieldType;
import com.baidu.bjf.remoting.protobuf.annotation.Protobuf;

import lombok.Data;

@Data
public class CardGroupInfo
{

  @Protobuf(description = "id")
  private String id;
  @Protobuf(description = "属于哪个玩家")
  private String pid;
  @Protobuf(description = "卡组名称")
  private String name;
  @Protobuf(description = "基地")
  private String baseInitId;
  @Protobuf(description = "卡牌组",fieldType=FieldType.STRING)
  private List<String> cardIds;
  
}
